### ISSUE TITLE
Allow docstring in NamedTuple subclasses

### ISSUE THREAD

**Issue by Stiivi** [python/mypy#3043]
MyPy will complain `Invalid statement in NamedTuple definition` when a doc string is added in the `NamedTuple` subclass:

```python
from typing import NamedTuple

class Thing(NamedTuple):
    """This is a thing"""
    value: int
````

Sibling issue: python/typing#404

**Comment by gvanrossum** [comment]
Oh, that should be easy to fix -- wanna make a PR?

**Comment by ilevkivskyi** [comment]
This will be fixed by #3081

**Comment by elliott-beach** [comment]
This was fixed and should be closed.
